src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,70 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import lotto.domain.constant.LottoConstant;
+import lotto.domain.exception.ExceptionMessage;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        this.numbers = new ArrayList<>(numbers);
+        Collections.sort(this.numbers);
+        validate(this.numbers);
+    }
+
+    private void validate(List<Integer> numbers) {
+        if (numbers.size() != LottoConstant.LOTTO_NUMBER_COUNT) {
+            throw new IllegalArgumentException(ExceptionMessage.INVALID_LOTTO_NUMBER_COUNT);
+        }
+
+        for (int i = 1; i < numbers.size(); ++i) {
+            if (Objects.equals(numbers.get(i - 1), numbers.get(i))) {
+                throw new IllegalArgumentException(ExceptionMessage.LOTTO_NUMBER_DUPLICATED);
+            }
+        }
+    }
```

Comment:
저는 적용하지 못했는데 Set 을 이용해서 로또 번호의 중복을 관리하는 분들도 많더라구요! 

```
public class Lotto {

    private final Set<Integer> numbers;

    public Lotto(Set<Integer> numbers) {
        validate(numbers);
        this.numbers = numbers;
    }

    public Set<Integer> getNumbers() {
        return new HashSet<>(numbers); // Return a copy to prevent modification outside of the class
    }

    private void validate(Set<Integer> numbers) {
        if (checkNumbersCount(numbers) || hasDuplicates(numbers)) {
            throw new IllegalArgumentException();
        }
    }

    private boolean checkNumbersCount(Set<Integer> numbers) {
        return numbers.size() != LottoConstants.NUMBER_OF_LOTTO_NUMBERS.getValue();
    }

    private boolean hasDuplicates(Set<Integer> numbers) {
        return numbers.size() != new HashSet<>(numbers).size();
    }
}

```
이런 느낌으로 개선해보는 방법이 있지 않을까 합니다 ㅎㅎ